From 84b050d060209eb692a341d3cf12059cc8d221ca Mon Sep 17 00:00:00 2001
From: Simone Esposito <31246543+DarkJoker360@users.noreply.github.com>
Date: Tue, 10 Nov 2020 11:32:48 +0100
Subject: [PATCH] Revert "Move webview to /product "

* Since there is no /product on Pie it causes crashes because it can't be found.
This reverts commit 60548e20c6aa97567ac51eec77c31414cd9bf2bc
---
 Android.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 566b02e..581462e 100644
--- a/Android.mk
+++ b/Android.mk
@@ -22,7 +22,6 @@ include $(CLEAR_VARS)
 
 LOCAL_MODULE := webview
 LOCAL_MODULE_CLASS := APPS
-LOCAL_PRODUCT_MODULE := true
 LOCAL_MULTILIB := both
 LOCAL_CERTIFICATE := $(DEFAULT_SYSTEM_DEV_CERTIFICATE)
 LOCAL_REQUIRED_MODULES := \
